libcomposefs/lcfs-writer.c

```diff
@@ -568,17 +592,41 @@ struct lcfs_node_s *lcfs_load_node_from_file(int dirfd, const char *fname,
 	ret->inode.st_size = sb.st_size;
 
 	if ((sb.st_mode & S_IFMT) == S_IFREG) {
-		if (sb.st_size != 0 && (buildflags & LCFS_BUILD_COMPUTE_DIGEST) != 0) {
-			int fd = openat(dirfd, fname, O_RDONLY | O_CLOEXEC);
+		bool compute_digest = (buildflags & LCFS_BUILD_COMPUTE_DIGEST) != 0;
+		bool inline_small = (buildflags & LCFS_BUILD_INLINE_SMALL) != 0;
+		bool do_digest = sb.st_size != 0 && compute_digest;
+		bool do_inline = sb.st_size > 0 && sb.st_size <= 64 && inline_small;
```

Comment:
It looks odd to me to use `sb.st_size != 0` on one line and then `sb.st_size > 0` in the next, when they're equivalent...

Maybe
```
bool is_zerosized = sb.st_size == 0;
bool do_digest = !is_zerosized && compute_digest;
bool do_inline = !is_zerosized && inline_small && sb.st_size <= LCFS_INLINE_SIZE;
```

or so?

Reply:
Yeah, that may be easier to read.